core/mavlink_system.h

```diff
@@ -19,7 +20,18 @@ namespace dronecore {
 class DroneCoreImpl;
 class PluginImplBase;
 
-class Device
+
+// This represents DroneCore client application which is a GCS.
+struct GCSClient {
```

Comment:
Please add in the comment or name what a `GCS` is.

Reply:
OK